### ISSUE TITLE
[Bug Report] Dropdown for studio shows limited options

### ISSUE THREAD

**Issue by michaelkrieger** [stashapp/stash-box#339]
**Describe the bug**
Entering a studio with a popular word matches each word individually and displays a finite number (25) of options.  If the desired studio is below the cut-off, you can never choose the correct studio and thus never properly enter the scene.

**To Reproduce**
1. Try to add a scene
2. Click in the studio box
3. Enter (for example) "girl cum"
4. 25 matches are displayed, in alphabetical order, ending long before the actual studio name, because anything including the word "Girl" is included.
5. Trying to put the studio in quotes does not produce different matches

Doing the same search within the "Studios" tab does find the correct studio on page 2 of 4.

**Expected behavior**
Possible options (in order)
1. Show exact/phrase matches first at the top of the list
2. Limit search results to require ALL words in the search as opposed to ANY of the words
3. Remove the arbitrary 25-result limit

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version 31

**Comment by peolic** [comment]
>Trying to put the studio in quotes does not produce different matches

works for me?

![image](https://user-images.githubusercontent.com/66393006/158699705-71451083-fea8-40f2-a7f3-8a01e6ff16cd.png)

**Comment by michaelkrieger** [comment]
> works for me?
You're right.  That didn't work when I did it a few days ago.  So quotes are the answer at this point.

Won't close it myself as I still remain that this should probably search words "and" instead of "or", or prioritize exact matches.

**Comment by InfiniteStash** [comment]
The studio name query was changed to be exact as part of the sqlc refactor.
